lib/kafo/kafo_configure.rb

```diff
@@ -307,9 +311,9 @@ def set_app_options
                             :default => false
       self.class.app_option ['-s', '--skip-checks-i-know-better'], :flag, 'Skip all system checks', :default => false
       self.class.app_option ['--skip-puppet-version-check'], :flag, 'Skip check for compatible Puppet versions', :default => false
-      self.class.app_option ['-v', '--verbose'], :flag, 'Display log on STDOUT instead of progressbar'
```

Comment:
I think we could keep `--verbose`/`-v` as an option for a while, that simply prints a message that it is now the default behavior.

Otherwise it could be confusing to an user to try to use `--verbose` and receive only an error that the option doesn't exist.